src/core/src/FabricPrepareRenderResources.cpp

```diff
@@ -56,22 +56,10 @@ struct TileLoadThreadResult {
 };
 
 bool hasBaseColorTextureGltf(const FabricMesh& fabricMesh) {
-    return fabricMesh.material->getMaterialDefinition().hasBaseColorTextures() &&
+    return fabricMesh.material != nullptr && fabricMesh.material->getMaterialDefinition().hasBaseColorTexture() &&
            fabricMesh.materialInfo.baseColorTexture.has_value();
```

Comment:
Bug fix. The `nullptr` check prevent against a crash if you're using a custom tileset material but omniverse can't locate it (the full explanation for why this happens is a bit longer).